src/filters/simple-filter/filter-component.js

```diff
@@ -18,8 +18,7 @@ angular.module('patternfly.filters').component('pfFilter', {
     };
 
     function filterExists (filter) {
-      var foundFilter = _.find(ctrl.config.appliedFilters, {title: filter.title, value: filter.value});
-      return foundFilter !== undefined;
+      return angular.isUndefined(_.find(ctrl.config.appliedFilters, {title: filter.title, value: filter.value}));
```

Comment:
@jeff-phillips-18 the logic is a bit 🐻 , but `isUndefined` maintains existing logic, you'll notice on line 37, the `!` is removed  my thought here was to wholly encapsulate the check in the function rather than spreading the logic in two places (though maybe its less intuitive now?)

(if changed to `isDefined`, filter does not work unless line 37 is restored to be negated, the first gif shows it working just fine)

SO I can do whatever ya want here, restore it to original orrr change to isundefined and restore line 37...